src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/framework/api/summaries/ResourceUtil.java

```diff
@@ -35,6 +35,9 @@
   public static final Resource HEAP_MAX_SIZE = Resource.newBuilder()
           .setResourceEnum(ResourceEnum.HEAP)
           .setMetricEnum(MetricEnum.HEAP_MAX).build();
+  public static final Resource HEAP_USAGE = Resource.newBuilder()
+          .setResourceEnum(ResourceEnum.HEAP)
+          .setMetricEnum(MetricEnum.HEAP_USAGE).build();
   public static final Resource OLD_GEN_HEAP_USAGE = Resource.newBuilder()
```

Comment:
This is a good change but I am thinking since we already have the `OLD_GEN_HEAP_USAGE` can we just use that ? That would mean we can get away with touching less files. 
The thing about accounting for full heap is that, not all the bytes are active and some of them will be reclaimed in the next gc. thoughts ?